websocket/hd_auth.go

```diff
@@ -22,7 +22,7 @@ func (ws *WebsocketServer) loginHandler(logger *log.Entry, msg *websocket.Messag
 	ok = (ws.secret == secret)
 	if ok {
 		ws.loggedIn[msg.Session] = true
-		logger.Debug("done")
+		logger.Debug("login done")
```

Comment:
unnessary 
https://github.com/FreifunkBremen/freifunkmanager/blob/393caf55f8491fe6919a59818ec1dbb2a00b478e/websocket/handler.go#L13-L19
 creates a subject field - dynamicly 

text should nice parseable for possible interpration later